src/dotnet-aspnet-codegenerator/Program.cs

```diff
@@ -231,14 +230,13 @@ internal int Execute(string[] args, bool isNoBuild)
             var depsFile = Path.Combine(targetDir, context.DepsFile);
 
             string dotnetCodeGenInsideManPath = string.Empty;
-            
-            if (IsNetCoreAppFramework(frameworkToUse))
+            if (IsNetCoreAppFramework(frameworkToUse) || IsNet5OrAbove(frameworkToUse))
```

Comment:
Why separate check for IsNet5OrAbove? If you are checking for the TFI, then net5.0 & above would also return true for the .NETCore check. 

The net5 check is only needed if you have just the shortname and not the TFI

Reply:
TFI for net5.0 is .NETFramework and netcoreapp is .NETCoreApp. We do something similar in WTE. Need to have checks for both separately.

Reply:
TFI for net5.0 should be .NETCoreApp. What we do in WTE is only if you start with the shortname. That is different. Let me know when you have some time so that we can chat in more detail on this.